benefits/eligibility/views.py

```diff
@@ -7,30 +7,40 @@
 from django.template.response import TemplateResponse
 from django.urls import reverse
 from django.utils.decorators import decorator_from_middleware
+from django.views import View
 
 from benefits.routes import routes
 from benefits.core import recaptcha, session
 from benefits.core.middleware import AgencySessionRequired, RecaptchaEnabled, FlowSessionRequired
+from benefits.core.mixins import AgencySessionRequiredMixin, RecaptchaEnabledMixin
 from benefits.core.models import EnrollmentFlow
 from . import analytics, forms, verify
 
 TEMPLATE_CONFIRM = "eligibility/confirm.html"
 
 
-@decorator_from_middleware(AgencySessionRequired)
-@decorator_from_middleware(RecaptchaEnabled)
-def index(request):
+class IndexView(AgencySessionRequiredMixin, RecaptchaEnabledMixin, View):
     """View handler for the enrollment flow selection form."""
-    agency = session.agency(request)
-    session.update(request, eligible=False, origin=agency.index_url)
 
-    # clear any prior OAuth token as the user is choosing their desired flow
-    # this may or may not require OAuth, with a different set of scope/claims than what is already stored
-    session.logout(request)
+    template_name = "eligibility/index.html"
```

Comment:
Should this template name be defined on line 19, under `TEMPLATE_CONFIRM`, for consistency?

Reply:
I think these global variables for template names were an artifact of having view functions. With class-based views, one of the main benefits is that everything is more cohesive and together, so I don't think we need to keep using these global variables for template names.